slices/util-linux.yaml

```diff
@@ -0,0 +1,130 @@
+package: util-linux
+
+essential:
+  - util-linux_copyright
+
+slices:
+  bins:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libcrypt1_libs
+      - libmount1_libs
+      - libpam0g_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libsystemd0_libs
+      - libtinfo6_libs
+      - libudev1_libs
+      - libuuid1_libs
+      - util-linux_generated
+      - util-linux_pam-profile
+      - zlib1g_libs
+    contents:
+      /usr/bin/addpart:
+      /usr/bin/choom:
+      /usr/bin/chrt:
+      /usr/bin/delpart:
+      /usr/bin/dmesg:
+      /usr/bin/fallocate:
+      /usr/bin/findmnt:
+      /usr/bin/flock:
+      /usr/bin/getopt:
+      /usr/bin/hardlink:
+      /usr/bin/i386: {arch: [i386, amd64]}
+      /usr/bin/ionice:
+      /usr/bin/ipcmk:
+      /usr/bin/ipcrm:
+      /usr/bin/ipcs:
+      /usr/bin/last:
+      /usr/bin/lastb:
+      /usr/bin/linux32:
+      /usr/bin/linux64:
+      /usr/bin/lsblk:
+      /usr/bin/lscpu:
+      /usr/bin/lsipc:
+      /usr/bin/lslocks:
+      /usr/bin/lslogins:
+      /usr/bin/lsmem:
+      /usr/bin/lsns:
+      /usr/bin/mcookie:
+      /usr/bin/mesg:
+      /usr/bin/more:
+      /usr/bin/mountpoint:
+      /usr/bin/namei:
+      /usr/bin/nsenter:
+      /usr/bin/partx:
+      /usr/bin/ppc: {arch: [ppc64el]}
+      /usr/bin/ppc32: {arch: [ppc64el]}
+      /usr/bin/ppc64: {arch: [ppc64el]}
+      /usr/bin/prlimit:
+      /usr/bin/rename.ul:
+      /usr/bin/resizepart:
+      /usr/bin/rev:
+      /usr/bin/s390: {arch: [s390x]}
+      /usr/bin/s390x: {arch: [s390x]}
+      /usr/bin/setarch:
+      /usr/bin/setpriv:
+      /usr/bin/setsid:
+      /usr/bin/setterm:
+      /usr/bin/su:
+      /usr/bin/taskset:
+      /usr/bin/uclampset:
+      /usr/bin/unshare:
+      /usr/bin/utmpdump:
+      /usr/bin/wdctl:
+      /usr/bin/whereis:
+      /usr/bin/x86_64: {arch: [amd64]}
+      /usr/sbin/agetty:
+      /usr/sbin/blkdiscard:
+      /usr/sbin/blkid:
+      /usr/sbin/blkzone:
+      /usr/sbin/blockdev:
+      /usr/sbin/chcpu:
+      /usr/sbin/chmem:
+      /usr/sbin/ctrlaltdel:
+      /usr/sbin/findfs:
+      /usr/sbin/fsck*:
+      /usr/sbin/fsfreeze:
+      /usr/sbin/fstrim:
+      /usr/sbin/getty:
+      /usr/sbin/isosize:
+      /usr/sbin/ldattach:
+      /usr/sbin/mkfs*:
+      /usr/sbin/mkswap:
+      /usr/sbin/pivot_root:
+      /usr/sbin/readprofile:
+      /usr/sbin/rtcwake:
+      /usr/sbin/runuser:
+      /usr/sbin/sulogin:
+      /usr/sbin/swaplabel:
+      /usr/sbin/switch_root:
+      /usr/sbin/wipefs:
+      /usr/sbin/zramctl:
+
+  generated:
```

Comment:
why `generated`?

Reply:
This is ported from #330. That PR is almost ready, so even if it got changed afterwards it shouldn't affect this PR much.